classes/eventobservers.php

```diff
@@ -24,8 +24,12 @@
 
 namespace local_boostnavigation;
 
+use cache;
+
 defined('MOODLE_INTERNAL') || die();
 
+require_once(__DIR__ . '/../locallib.php');
```

Comment:
do you need this?

Reply:
Hey Brendan,

Added this line back to pass the Travis.

Thanks,
Anu